android/cli/commands/_buildModule.js

```diff
@@ -269,7 +269,7 @@ AndroidModuleBuilder.prototype.initialize = function initialize(next) {
 	this.templatesDir = path.join(this.platformPath, 'templates', 'build');
 	this.moduleIdSubDir = this.manifest.moduleid.split('.').join(path.sep);
 
-	['assets', 'documentation', 'example', 'platform', 'Resources'].forEach(function (folder) {
+	['assets', 'documentation', 'example', 'platform', 'Resources', 'hooks'].forEach(function (folder) {
```

Comment:
There's no point including `'hooks'` here since you're overriding it on line 291.